packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-code-deploy.js.snapshot/aws-cdk-codepipeline-codedeploy.template.json

```diff
@@ -59,6 +59,9 @@
       "Arn"
      ]
     },
+    "AlarmConfiguration": {
+     "Enabled": false
+    },
```

Comment:
A user upgrading to a new version will see this in their diff, which we try to avoid. I don't think there's a way around this here, and we can allow it for this change.

Reply:
This is a breaking change (however minor) so we need a feature flag for it. Take a look at #20765 for an example of adding a feature flag.